README.md

```diff
@@ -4,8 +4,8 @@
 
 disnake
 =======
-
 <p align="center">
+    <img src="https://img.shields.io/github/actions/workflow/status/DisnakeDev/disnake/lint-test.yml?branch=master"></img>
```

Comment:
```suggestion
    <img src="https://img.shields.io/github/actions/workflow/status/DisnakeDev/disnake/lint-test.yml?branch=master&style=flat-square"></img>
```
`flat-square` makes this match the other badges better (old vs new):
![image](https://github.com/DisnakeDev/disnake/assets/8530778/4ee6cded-fa94-4a64-8655-e713fec803f6)